pkg/vpcmodel/explainabilityConnectivity.go

```diff
@@ -48,8 +69,8 @@ type srcDstDetails struct {
 	crossVpcRules  []RulesInTable  // cross vpc (only tgw at the moment) prefix rules effecting the connection (or lack of)
 	// there could be more than one connection effecting the connection since src/dst cidr's may contain more than one AP
 
-	// loadBalancerDenyEgressRule - true if src is pip, and dst is not pool member:
-	loadBalancerRule loadBalancerDenyEgressRule
+	// loadBalancerRule - the lb rule affecting this connection.
```

Comment:
please explain shortly in what case this rule is relevant and why it is unique - namely, why there can not be more than one LB rule relevant